File: 프로그래머스/lv0/120844. 배열 회전시키기/배열 회전시키기.swift
import Foundation

func solution(_ numbers:[Int], _ direction:String) -> [Int] {
    var answer = numbers
    if direction == "right" {
        answer.insert(numbers[numbers.count - 1], at: 0)
        answer.remove(at: numbers.count)
    }
    if direction == "left" {
        answer.append(numbers[0])
        answer.remove(at: 0)
    }
    return answer
}

File: 프로그래머스/lv0/120844. 배열 회전시키기/배열 회전시키기.js
function solution(numbers, direction) {
    if (direction === 'left') {
        const first = numbers.shift();
        numbers.push(first);
        return numbers;
    }
    if (direction === 'right') {
        const last = numbers.pop();
        numbers.unshift(last);
        return numbers;
    }
}